new/LIT6910MarkFiveWoman.xml

```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6910MarkFiveWoman" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Mark 5,25-29</title>
```

Comment:
I would have a descriptive title
Miracle of Jesus healing the bleeding woman
as t1
and put the passage in the abstract or as t2
On referring to Bible text passages see https://betamasaheft.eu/Guidelines/?q=passage&id=referencing-the-text

Reply:
Yes, I updated the titles accordingly. I understood, that I should give "Mark 5,25-29" as t2, but not the text in itself.

In Perseus, I found the corresponding Greek text as follows:
Kata Markon 5, 24-29
<img width="449" alt="Screenshot 2024_kata Markon" src="https://github.com/BetaMasaheft/Works/assets/40291787/81c7372a-4427-4f1d-a781-e4bec670f526">

Following the quidelines, I still feel not competent enough to refer to this passage in the correct way. I tried as flollows: 

```
  <div type="edition">
                <div type="textpart" xml:lang="gez" subtype="incipit">
                    <cit>
                        <quote xml:lang="gez">ወመጽአት፡ ብእሲት፡ እንተ፡ ደም፡ ይውሕዛ፡ እምዐሠርቱ፡ ወክልኤቱ፡ ክረምት። ወብዙኅ፡ አሕመምዋ፡ ብዙኃን፡ ዐቀብተ፡ ሥራይ፡ 
                            ወአስተዋፅአት፡ ኵሎ፡ ንዋያ፡ ወአኅለቀት፡ ወአልቦ፡ ዘበቍዐ፡ ዘእንበለ፡ ዘአልሐማ፡ ዳእሙ፡ ፈድፋደ። ወሰሚዓ፡ በእንተ፡ ኢያሱስ፡ መጽአት፡ ወቦአት፡ ማአከለ፡ ሰብእ፡ ወገሰሰት፡
                            ጽንፈ፡ ልብሱ፡ በድኅሬሁ። እንዘ፡ ትብል፡ እምከመ፡ ገሰስኩ፡ ጽንፈ፡ ልብሱ፡ አሐዩ። ወሶቤሃ፡ ነጽፈ፡ ነቅዐ፡ ደማ፡ ወአንከረት፡ ርእሳ፡ ከመ፡ ሐይወት፡ እምዳዌሃ።
                        </quote>
                        <ref cRef="betmas:LIT6783MarkFive">Mark 5:25-29</ref>
                        <ref cRef="urn:cts:greekLit:tlg001.perseus-grc2:5.25-29">Mark 5, 25-29</ref>
                    </cit>
                </div>

```

I hope, this is OK or not too far from it.